website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md

```diff
@@ -16,64 +16,64 @@ If you are using Appsmith Business Edition, with the Helm chart from `helm.appsm
    appsmithctl backup
    ```
 
-3. Once this is finished, you should have a backup archive generated at a location like this. This is the Appsmith backup archive.
+3. Once the backup process is complete, the backup archive is available at a location like this:
 
    ```
    /appsmith-stacks/data/backup/appsmith-backup-2022-10-24T07-09-56.930Z.tar.gz
    ```
 
-4. Exit out of the shell with:
+4. To exit the shell, run the following command:
```

Comment:
Can we undo the change here as well please? This reads like it's just an instruction on _how_ to do something. Not like an instruction to just go do it. We're not informing them on how to exit the shell, we're telling them to exit the shell. Something like `Please exit the shell, by running the following command:` or something.

I see a lot of these wordings, can we change them please?

Reply:
We don't use words like please, should, could, would, and so on in docs. What do you think if I reword it as below:

> Run the below command to exit the shell.